test/tests/functional/pbs_mom_hook_sync.py

```diff
@@ -120,7 +120,7 @@ def tearDown(self):
         self.momB.signal("-CONT")
         TestFunctional.tearDown(self)
 
-    def test_1(self):
+    def test_mom_svr_mom_hooksync_with_susp_resume(self):
```

Comment:
I feel the test name should be "test_momhook_to_serverhook_with_resume". since the test is inside momhooksync test suite we do not need to explicitly mention in the test name.